client/blocks/reader-full-post/index.jsx

```diff
@@ -189,7 +189,7 @@ export class FullPostView extends React.Component {
 								siteIcon={ get( site, 'icon.img' ) }
 								feedIcon={ get( feed, 'image' ) }
 								siteName={ post.site_name }
-								siteUrl= { post.site_URL }
+								feedUrl= { get( feed, 'feed_URL' ) }
```

Comment:
Right you are.

@blowery noted in Slack that there are two types of sites that may have a blog ID but not a feed ID:
- Jetpack sites
- wpcom sites with no subscriptions (there aren't many, but they certainly exist)